.github/workflows/deploy.sh

```diff
@@ -37,8 +37,8 @@ if [ -n "$selection" ]; then
     then
     destination=""
     branch=""
-    az acr run --cmd "acr purge --filter 'get-a-quote:.*' --ago 15d" --registry mdm-development /dev/null
-    az acr run --cmd "acr purge --filter 'get-a-quote:.*' --ago 15d" --registry mdm-production /dev/null
+    az acr run --cmd "acr purge --filter 'get-a-quote:.*' --ago 15d" --registry "${env.APIM_MDM_ACR}" /dev/null
+    az acr run --cmd "acr purge --filter 'get-a-quote:.*' --ago 15d" --registry "${env.APIM_MDM_ACR}" /dev/null
```

Comment:
Aren't this line with the one above the same?

Reply:
Yes it is, apologies have removed.